### PR TITLE
Lots up updates in this PR - greatly improved speed of TOP ACCOUNTS, fixed POOLS page, much more.

### PR DESCRIPTION
**Pull request by pir8radio**
guess i forgot to do a PR for the tx filter stuff a while back. unfortunatuly the transaction changes to the node API have made explorer transactions REALLY slow and fail due to timeouts quite often.. hope @ohager can fix that up soon.

### CODE REVIEW

File: scan/views/pools.py
Comment (pir8radio) [inline scan/views/pools.py:1]: this change filters duplicate pools from the list.   now prints to single pool list as well.

File: scan/views/json.py
Comment (pir8radio) [inline scan/views/json.py:1]: This change adds an API call for all nodes..  Handy if anyone wants to get an easy list of all nodes in json format. requested by user.

File: scan/views/filters/transactions.py
Comment (pir8radio) [inline scan/views/filters/transactions.py:1]: this is part of the tx filtering changes.    blank was 000 but returned a real account, changed to 99 to limit false returns.

File: scan/templates/base.html
Comment (pir8radio) [inline scan/templates/base.html:1]: upgraded explorer font awesome to 6.5.2   i kept in the 6.2.1 link as not to break any existing installs..  but please upgrade your fontawesome-free-web folder to 6.5.2 delete 6.2.1

**Review by pir8radio — COMMENTED** [review]
added comments as to what the changes do.

File: config/settings.py [hunk 1]
```diff
@@ -257,6 +257,8 @@
 # UA-XXXXXXXXX-X
 GOOGLE_TRACKING_ID = os.environ.get("GOOGLE_TRACKING_ID")
 
+IPFS_GATEWAY = os.environ.get("IPFS_GATEWAY", "https://ipfs.io/ipfs/")
```
Comment (frankTheTank72) [inline config/settings.py:260]: maybe we need anoother gateway soon

File: config/settings.py [hunk 2]
```diff
@@ -257,6 +257,8 @@
 # UA-XXXXXXXXX-X
 GOOGLE_TRACKING_ID = os.environ.get("GOOGLE_TRACKING_ID")
```
Comment (frankTheTank72) [inline config/settings.py:258]: i think google is not used anymore- we should remove that

File: scan/templates/base.html
Comment (frankTheTank72) [reply scan/templates/base.html:1]: done - upgraded - we can remove those old links

**Review by frankTheTank72 — APPROVED** [review]

File: config/settings.py (hunk 1 above)
Comment (pir8radio) [reply config/settings.py:260]: > maybe we need another gateway soon

yes i made this a .env value   just add it to the .env and make it whatever you want.  
IPFS_GATEWAY=https://newgateway.com/ipfs/               
right now it defaults to ipfs.io/ipfs/ if no value is in .env

File: config/settings.py (hunk 2 above)
Comment (pir8radio) [reply config/settings.py:258]: i use it, but yes understood, most people don't use google analytics. we can remove.    I just use it to aid in changes and troubleshooting.    
image

**Pull request merged**